tensorflow_probability/python/stats/sample_stats.py

```diff
@@ -469,6 +476,81 @@ def covariance(x,
 
     return cov
 
+CovarianceState = collections.namedtuple('CovarianceState',
+                                         'num_samples, mean, comoment, dtype')
+def init_running_covariance(shape, dtype=tf.float32):
+  """Initialize the Running Covariance State.
+
+  Args:
+    shape: Python `Tuple` representing the shape of incoming samples.
+    dtype: Dtype of incoming samples and the resulting statistics.
+      By default, the dtype is `tf.float32`
+
+  Returns:
+    state: `CovarianceState` representing a stream of no inputs
+  """
+  return CovarianceState(
+      num_samples=0,
+      mean=tf.zeros(shape),
+      comoment=tf.zeros(shape + shape),
+      dtype=dtype,
+    )
+
+def update_running_covariance(curr_state, new_sample):
+  """Update the streaming statistics of the `VarianceState` with a new sample
```

Comment:
You mean, `CovarianceState`?

Reply:
Ah yes, thanks for the catch; the Docstrings were updated quite late at night and this slipped.